Cargo.toml

```diff
@@ -29,7 +29,7 @@ http-body = "0.3.1"
 httparse = "1.0"
 h2 = "0.2.2"
 itoa = "0.4.1"
-log = "0.4"
+tracing = { version = "0.1", default-features = false, features = ["log", "std"] }
```

Comment:
It would be good to expose the `log` feature optionally (though it should default to on so we don't break existing code that expects `log` records emitted by Hyper), so that `tracing` users who don't need `log` can disable it. Otherwise, depending on Hyper will _always_ enable the `log` feature, even when users don't need it.

Reply:
hmm, solid point. will fix.

Reply:
Something to consider, though, is that if the logging enabled becomes an optional feature, then those who have `default-features = false` will have the logs disabled when they upgrade. 

(I find cargo's default-features to be annoying in this regard, but what we gonna do.)

Reply:
Darn, that's an annoying thing. Are you okay with breaking those people in a minor release? Or more generally, how _should_ this be handled?